src/test/regress/citus_tests/upgrade/pg_upgrade_test.py

```diff
@@ -56,7 +57,24 @@ def perform_postgres_upgrade(
                 "--new-datadir",
                 abs_new_data_path,
             ]
-            subprocess.run(command, check=True)
+            proc = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
I think it would make more sense to store this file as an artifact similar to:

https://github.com/citusdata/citus/blob/6f5a343ff48489608d4b58e370c6324d39bbb2d0/.circleci/config.yml#L403-L405

That way we can check/download it from the [artifacts tab](https://app.circleci.com/pipelines/github/citusdata/citus/19092/workflows/88d885cb-8e35-43e0-9f0e-755e18b2e901/jobs/443456/artifacts)

We can add this `store_artifact` step to pg_upgrade jobs, I expect it will be 3-4 additional lines to the config file. We can even store the whole `tmp_upgrade` folder so that we have even more files available as artifacts(assuming they are fast to upload).

Storing the whole `tmp_upgrade` folder could look something like this:

```yaml
      - store_artifacts:
          name: 'Store upgrade artifacts'
          path: src/test/regress/tmp_upgrade
```

and the postgres logs would be in `newData` folder:

![image](https://user-images.githubusercontent.com/17179875/141157268-6941148e-a4c4-480c-9c53-d1ee4c0c16de.png)

Reply:
done